src/components/Dialogs/MessageForm.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import {Field, reduxForm} from 'redux-form';
+
+import {Textarea} from '../common/FormsControl';
+import {maxLengthCreator, required} from '../../utils/validators';
+
+import styles from './styles.module.css';
+
+const maxLength50 = maxLengthCreator(50);
+
+const AddMessageForm = props => (
+  <form onSubmit={props.handleSubmit}>
+    <div className={styles.form}>
+      <Field component={Textarea} validate={[required, maxLength50]} name="newMessageBody" placeholder="Enter your message" className={styles.messageInput} />
+    </div>
+    <div>
+      <button>Send</button>
+    </div>
+  </form>
+);
+
+const AddMessageFormRedux = reduxForm({form: 'dialogAddMessageForm'})(AddMessageForm);
```

Comment:
I don't think you need `AddMessageFormRedux`
you can export `reduxForm({form: 'dialogAddMessageForm'})(AddMessageForm)` directly